views/default/lightbox/elgg-colorbox-theme/colorbox.css

```diff
@@ -127,7 +123,7 @@
 }
 
 #cboxLoadingOverlay {
-	background: transparent url(graphics/ajax_loader_bw.gif) no-repeat center center;
+	background: transparent url(../../graphics/ajax_loader_bw.gif) no-repeat center center;
```

Comment:
don't understand why `graphics/ajax_loader` is not loading `views/default/graphics/ajaxloader` then...